qdevices/qnetd-cluster-list.c

```diff
@@ -61,10 +61,17 @@ qnetd_cluster_list_find_by_name(struct qnetd_cluster_list *list,
 	return (NULL);
 }
 
+static int
+client_node_id_cmp(const struct qnetd_client *entry1, const struct qnetd_client *entry2)
+{
+	return (entry1->node_id - entry2->node_id);
```

Comment:
Please replace this with `if (entry1->node_id < entry2->node_id) { return (-1); } else if (...`. I know it is longer and this substraction should work, but in general it doesn't work and it is source of quite a big bugs (https://marc.info/?l=openbsd-tech&m=141806297022265&w=2) and it is just better to not have it in codebase.

Reply:
Changed
Thanks!